### ISSUE TITLE
Calculate detection AUROC

### ISSUE THREAD

**Issue by jpmrs1313** [gudovskiy/cflow-ad#22]
In the line 334 of train file, the detection AUROC is calculated using  truth label and score label. 
Why the truth label is bolean and score label is a float? Im trying to replicate using the leather class and the score label values are between [0.97,2.7].

**Comment by gudovskiy** [comment]
@jpmrs1313 it is the nature of [auroc](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html) and other threshold-agnostic metrics. In your binary case, you can normalize [0.97, 2.7] to [0,1]
